src/main.cpp

```diff
@@ -62,6 +71,11 @@ int main(int argc, char *argv[]) {
 
     int result = DiscreteVerification::DiscreteVerification::run(*tapn, initialPlacement, query.get(), options);
 
+    if (options.getPrintBindings()) {
+        std::cout << output_stream->str();
+        delete output_stream;
```

Comment:
if you don't use new, you don't have to deallocate. An empty strinstream should be negligible in overhead.